apps/services/notifications/src/app/modules/notifications/dto/createNotification.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator'
+import { ApiProperty } from '@nestjs/swagger'
+import { IsNationalId } from '@island.is/nest/validators'
+
+export enum MessageTypes {
+  NewPostholfMessage = 'newPostholfMessage',
```

Comment:
Should this maybe not include icelandic in name?

Reply:
Again, RFC. I don't care one way or the other